digdag-ui/model.js

```diff
@@ -309,7 +309,7 @@ export class Model {
   }
 
   fetchProjectWorkflowSchedule (projectId: number, workflowName: string): Promise<*> {
-    return this.get(`schedules?project_id=${projectId}&workflow=${workflowName}`)
+    return this.get(`projects/${projectId}/schedules?project_id=${projectId}&workflow=${workflowName}`)
```

Comment:
`project_id` param is no longer needed as its part of the url as per
https://github.com/treasure-data/digdag/blob/e6726c07583cf046baea3d5273f42039cb55164b/digdag-server/src/main/java/io/digdag/server/rs/ProjectResource.java#L131